Formula/gdb.rb

```diff
@@ -12,6 +13,10 @@ class Gdb < Formula
     sha256 "4cde626aa5d32dde54d70bd531a06e65051e7ac7371f1970b6b9c838f565239c" => :high_sierra
   end
 
+  depends_on "xz" # required for lzma support
+
+  uses_from_macos "python@2"
```

Comment:
is this required for lzma?

Reply:
Nope, I just found that `gdb` uses it

Reply:
I think we should try to restrict `uses_from_macos "python@2"` for cases were it will never work with Python 3. I am trying to get rid of Python 2, and have still more than 60 formulae to migrate ... so please don't add more :)

For these cases, I try to depend on python@3.8 (which does not mean that it will necessarily use it, as it may silently pick up system python 2 ... the only way to know is to run the build on linuxbrew, where there is no python 2 ...)

Reply:
Ah, I see.
Switched it to `python@3.8` then